Libplanet/Blockchain/Policies/IBlockPolicy.cs

```diff
@@ -9,12 +10,23 @@ namespace Libplanet.Blockchain.Policies
     /// valid, and to suggest how difficult a <see cref="Block{T}.Nonce"/>
     /// for a <see cref="Block{T}"/> to be mined.
     /// </summary>
-    /// <typeparam name="T">An <see cref="IAction"/> type.  It should match
-    /// to <see cref="Block{T}"/>'s type parameter.</typeparam>
+    /// <typeparam name="TTxAction">An <see cref="IAction"/> type used for
+    /// <see cref="Transaction{T}"/>.  It should match to <see cref="Block{T}"/>'s type parameter.
+    /// </typeparam>
+    /// <typeparam name="TBlockAction">An <see cref="IAction"/> type used when mining.
```

Comment:
The explanation “used when mining” listens misleading to me, because even a node that does not mine anyway executes and renders this action.